mat/cholesky.go

```diff
@@ -503,9 +503,9 @@ func (c *Cholesky) SymRankOne(orig *Cholesky, alpha float64, x Vector) (ok bool)
 				// Multiply the extended factorization matrix by
 				// the Givens matrix from the left. Only
 				// the i-th row and x are modified.
-				blas64.Rot(n-i-1,
-					blas64.Vector{Data: umat.Data[i*stride+i+1 : i*stride+n], Inc: 1},
-					blas64.Vector{Data: work[i+1 : n], Inc: 1},
+				blas64.Rot(
+					blas64.Vector{N: n - i - 1, Data: umat.Data[i*stride+i+1 : i*stride+n], Inc: 1},
```

Comment:
I think this is the kind of case we were considering as well.

Reply:
Sorry, that is meant to be read as "I agree with you this is a net win"

Reply:
>The fact that the size is now doubled?

No, that we are passing the length twice.